app/controllers/skills_controller.rb

```diff
@@ -34,6 +34,15 @@ def update
     end
   end
 
+  def index
+    if params[:query]
+      @skills = Skill.search(params[:query]).order('name ASC')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.